apps/web/pages/settings/my-account/calendars.tsx

```diff
@@ -36,6 +36,16 @@ const SkeletonLoader = () => {
   );
 };
 
+const AddCalendarButton = () => {
+  return (
+    <>
+      <Button color="secondary" StartIcon={Icon.FiPlus} href={`${CAL_URL}/apps/categories/calendar`}>
```

Comment:
can you not just link to /apps/categories/calendar and next-js router picks it up correctly?